model/src/main/java/de/evoila/cf/broker/model/context/Context.java

```diff
@@ -0,0 +1,200 @@
+            switch (platform) {
+                case PLATFORM_CLOUDFOUNDRY:
+                case PLATFORM_KUBERNETES:
+                    break;
+                default:
+                    throw new IllegalArgumentException("Only Cloudfoundry and Kubernetes are supported Platforms.");
+            }
+        } else {
+            throw new IllegalArgumentException("Did not find a value for Platform!");
+        }
+
+        this.platform = platform;
+    }
+
+    public String getOrganizationGuid() {
+        return organizationGuid;
+    }
+
+    public void setOrganizationGuid(String organizationGuid) {
+        this.organizationGuid = organizationGuid;
+    }
+
+    public String getOrganizationName() {
+        return organizationName;
+    }
+
+    public void setOrganizationName(String organizationName) {
+        this.organizationName = organizationName;
+    }
+
+    public String getSpaceGuid() {
+        return spaceGuid;
+    }
+
+    public void setSpaceGuid(String spaceGuid) {
+        this.spaceGuid = spaceGuid;
+    }
+
+    public String getSpaceName() {
+        return spaceName;
+    }
+
+    public void setSpaceName(String spaceName) {
+        this.spaceName = spaceName;
+    }
+
+    public String getInstanceName() {
+        return instanceName;
+    }
+
+    public void setInstanceName(String instanceName) {
+        this.instanceName = instanceName;
+    }
+
+    public String getClusterId() {
+        return clusterId;
+    }
+
+    public void setClusterId(String clusterId) {
+        this.clusterId = clusterId;
+    }
+
+    public String getNamespace() {
+        return namespace;
+    }
+
+    public void setNamespace(String namespace) {
+        this.namespace = namespace;
+    }
+
+    @JsonAnyGetter
+    public Map<String, Object> getAdditionalFields() {
+        return additionalFields;
+    }
+
+    @JsonAnySetter
+    public void setAdditionalFields(String key, Object value) {
+        if (additionalFields == null) {
+            additionalFields = new HashMap<>();
+        }
+
+        this.additionalFields.put(key, value);
+    }
+
+    public void validateContextObject() {
+        switch (this.getPlatform()) {
+            case PLATFORM_CLOUDFOUNDRY:
+                validateCloudfoundryContextObject();
+                break;
+            case PLATFORM_KUBERNETES:
+                validateKubernetesContextObject();
+        }
+
+    }
+
+    private void validateKubernetesContextObject() {
+        if (fieldIsPresent(this.getNamespace()) && fieldIsPresent(this.getClusterId())) {
+            log.debug("Received a valid Kubernetes context object.");
+        } else {
+            throw new IllegalArgumentException("A Kubernetes Context object should contain the fields namespace and clusterId.");
+        }
+    }
+
+    private void validateCloudfoundryContextObject() {
+        if (fieldIsPresent(this.getOrganizationGuid()) &&
+                fieldIsPresent(this.getOrganizationGuid()) &&
```

Comment:
Just realized the doubled check for organization guid.

Reply:
fixed